src/main/resources/assets/forestry/lang/en_us.lang

```diff
@@ -1859,6 +1859,11 @@ for.config.butterfly.entities.pollination.comment=Allow butterflies to pollinate
 for.config.backpacks.item.stacks.format=Add itemStacks for the %s's backpack here in the format 'modid:name:meta'. For wildcard metadata the format is 'modid:name'.
 for.config.backpacks.ore.dict.format=Add ore dictionary names for the %s's backpack here in the format 'oreDictName'. Regular expressions may also be used.
 
+for.config.crates.items.accepted.comment=Added items
```

Comment:
t would be good if you could add more information to the comments of the config options. Things like how an entry has to look like or a list of all entries that can be blacklisted. Added / Blacklisted items is not much information for a user that sees the config for the first time.